src/main/java/Bob/Storage.java

```diff
@@ -0,0 +1,55 @@
+package bob;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+import bob.task.Task;
+
+class Storage {
+
+    /**
+     * Saves the tasks to a file.
+     *
+     * @param tasks
+     * @throws IOException
```

Comment:
Please add a description of the parameters. I noticed the same issue in several other places below too.